app/presenters/root_browse_page_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class RootBrowsePagePresenter
+
+  def render_for_publishing_api
+    {
+      content_id: "8413047e-570a-448b-b8cb-d288a12807dd",
+      format: "mainstream_browse_page",
+      title: "Browse",
+      locale: 'en',
+      public_updated_at: Time.now.iso8601,
```

Comment:
It's not possible for that to happen, because there's always at least one top level page (the page that's being published!).  I'd suggest raising an error in this case, rather than trying to handle it.